### PR TITLE
[NEW] RI feature doc

### PR DESCRIPTION
**Pull request by nermiller**
Adds doc for new RI features: database analysis, Redis streams support, search capabilities, bulk actions, and Slow Log

### CODE REVIEW

**Comment by nermiller** [comment]
@ViktarStarastsenka please review. Thank you!

File: docs/stack/insight/_index.md [hunk 1]
```diff
@@ -32,11 +32,15 @@ Browse, filter and visualize your key-value Redis data structures.
 * CRUD support for [RedisJSON](https://oss.redis.com/redisjson/)
 * Group keys according to their namespaces
 
-<img src="/docs/stack/insight/images/Browser.png">
+  <img src="/docs/stack/insight/images/Browser.png">
+
+* View, validate, and manage your key values using new formatters that prettify and highlight data in different formats (for example, Unicode, JSON, MessagePack, HEX, and ASCII)
+
+  <img src="/docs/stack/insight/images/data_formatting.png">
```
Comment (ViktarStarastsenka) [inline docs/stack/insight/_index.md:39]: @nermiller , let's use new images with higher resolution.
Please let me know if you need them.

Comment (nermiller) [reply docs/stack/insight/_index.md:39]: @ViktarStarastsenka yes, please, point me to them. Thanks!

Comment (K-Jo) [reply docs/stack/insight/_index.md:39]: View, validate, and manage your key values in a human-readable format using formatters that prettify and highlight data in different formats (for example, Unicode, JSON, MessagePack, HEX, and ASCII) in the Browser tool.

**Review by K-Jo — CHANGES_REQUESTED** [review]

File: docs/stack/insight/_index.md [hunk 2]
```diff
@@ -19,24 +19,28 @@ RedisInsight is a powerful tool for visualizing and optimizing data in Redis or
...
 Browse, filter and visualize your key-value Redis data structures.
-* [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) support for Lists, Hashes, Strings, Sets, Sorted Sets 
+* [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) support for lists, hashes, strings, sets, sorted sets, and streams 
 * CRUD support for [RedisJSON](https://oss.redis.com/redisjson/)
 * Group keys according to their namespaces
 
-<img src="/docs/stack/insight/images/Browser.png">
+  <img src="/docs/stack/insight/images/Browser.png">
+
+* View, validate, and manage your key values using new formatters that prettify and highlight data in different formats (for example, Unicode, JSON, MessagePack, HEX, and ASCII)
```
Comment (nermiller) [inline docs/stack/insight/_index.md:37]:
```suggestion
* View, validate, and manage your key values in a human-readable format using formatters that prettify and highlight data in different formats (for example, Unicode, JSON, MessagePack, HEX, and ASCII) in the Browser tool.
```

File: docs/stack/insight/_index.md [hunk 3]
```diff
@@ -53,13 +57,51 @@ The CLI is accessible at any time within the application.
...
+
+Create and manage streams by adding, removing, and filtering entries per timestamp. To see and work with new entries, enable and customize the automatic refresh rate.
+
+View and manage the list of consumer groups. See existing consumers in a given consumer name as well as the last messages delivered to them. Inspect the list of pending messages, explicitly acknowledge the processed items, or claim unprocessed messages via RedisInsight.
+
+<img src="/docs/stack/insight/images/streams.png">
+
+### Search capabilities
+
+If you're using Redis or Redis Stack's indexing, querying, or full-text search capabilities, RedisInsight provides UI controls to quickly and conveniently run search queries against a preselected index. You can also create a secondary index of your data in a dedicated pane.
```
Comment (K-Jo) [inline docs/stack/insight/_index.md:87]: this is not correct.  It's only for Redis stacks' indexing capabilities. need to go back to check the blog now

**Review by K-Jo — CHANGES_REQUESTED** [review]

File: docs/stack/insight/_index.md (hunk 3 above)
Comment (nermiller) [inline docs/stack/insight/_index.md:87]:
```suggestion
If you're using Redis Stack's indexing, querying, or full-text search capabilities, RedisInsight provides UI controls to quickly and conveniently run search queries against a preselected index. You can also create a secondary index of your data in a dedicated pane.
```

Comment (nermiller) [reply docs/stack/insight/_index.md]: updated!

**Review by K-Jo — APPROVED** [review]

**Pull request merged**